doctrine/phpcr-bundle/1.3/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+    "bundles": {
+        "Doctrine\\Bundle\\PHPCRBundle\\DoctrinePHPCRBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/",
+        "src/": "%SRC_DIR%/"
```

Comment:
Can you explain why we need to create a `Document` folder?

Reply:
I've created `Document` folder to be compliant with doc http://symfony.com/doc/master/cmf/bundles/phpcr_odm/models.html

Reply:
If you consider is not necessary for this recipe i'll remove it.

Reply:
I'm not familiar with the CMF, that is why Im asking questions =)

Am I required to put the documents in src/Document or could I as easily put them in src/Foo/Bar/Baz?

Reply:
You have right. I remove this part.